@here/harp-materials/lib/MapMeshMaterials.ts

```diff
@@ -1199,6 +1204,20 @@ export class MapMeshStandardMaterial extends THREE.MeshStandardMaterial
     set extrusionRatio(value: number) {
         // to be overridden
     }
+    /**
```

Comment:
Not needed now

Reply:
as mentioned, it needs to stay unfortunately. I wasn't able to work around it.